### DISCUSSION TITLE
IPXE ERROR 040EE119

### DISCUSSION THREAD

**Discussion by Ultimo86** [ipxe/ipxe#978] [General]
Hi all, 

I have a VNF on Openstack VIM : 

_python-novaclient (14.0.0)_
_openstack 3.17.0_

and when there's the stop&start or reboot nova compute, I have the 040EE119 error : 

_[1miPXE 1.0.0+ (b878)[0m -- Open Source Network Boot Firmware -- [36mhttp://ipxe.org[0m
Features: DNS HTTP TFTP ELF PXE bzImage Menu PXEXT
Count: 0 (rebooting at 10)
Configuring (net0 00:80:37:0e:09:22)...............Error 0x040ee119 (http://ipxe.org/040ee119)
Count: 1 (rebooting at 10)
Configuring (net0 00:80:37:0e:09:22)...............Error 0x040ee119 (http://ipxe.org/040ee119)
Count: 2 (rebooting at 10)
Configuring (net0 00:80:37:0e:09:22)...............Error 0x040ee119 (http://ipxe.org/040ee119)_

I checked the dhcpd.conf present on VNF, but it's ok and besides this issue is present only when there's the stop&start or reboot nova compute. Infact when I perform the server reboot through the VNF, it' ok : 

_[1miPXE 1.0.0+ (b878)[0m -- Open Source Network Boot Firmware -- [36mhttp://ipxe.org[0m
Features: DNS HTTP TFTP ELF PXE bzImage Menu PXEXT
Count: 0 (rebooting at 10)
Configuring (net0 00:80:37:0e:09:22)...... ok

ip.............. ....... 169.254.8.146
dhcp-hostname... 012.... eqm02s09p2
user class...... 077.... ipxe-virt

gateway......... 003.... 169.254.127.254
next-server..... 066.... 169.254.1.13
filename........ 067.... /tmp/DPE_CORE/SCs/sc.1/LoadUnits/BootFiles/virt/linux/kernel/vswlc/pxelinux.0_

We have some explanation about this issue ? Should I still investigate on VNF ? 
It seems that the DHCP server doesn't respond to the DISCOVER messages. 

Thanks in advance 

BR

**Comment by NiKiZe** [comment]
Check if any dhcp traffic comes out of the VM?
Maybe there is an issue with the state of the nic after being used in your OS and that isn't properly reset for the second boot?

**Reply by Ultimo86** [reply]
Unfortunately I'm not able to press Ctrl-B for the iPXE command line infact its seems that Ctrl-B doesn't work...

**Reply by NiKiZe** [reply]
If you have any embedded script that will "disable" ctrl-b.
But if you see the prompt you should be able to use it.

But to debug this issue, i would probable try to use a iPXE with embedded script that does the debug part.

However the checking dhcp part would be done on the host, using tolls such as tcpdump.

**Comment by Ultimo86** [comment]
>However the checking dhcp part would be done on the host, using tolls such as tcpdump.

in fact that's exactly what I asked the customer which manage the host. 
The VNF is guest.
